roles/ansible/tower/config-ansible-tower-ocp/README.md

```diff
@@ -29,6 +32,9 @@ The variables used to install Ansible Tower on OpenShift are outlined in the tab
 |openshift_skip_tls_verify| Should installer skip TLS verifcation of Openshift API|no|false|
 |openshift_pg_emptydir|Flag for Postgre to use EmptyDir for storage(not recommended for Production)|no|true|
 |openshift_pg_pvc_name|Persistent Volume Claim to be used for PostgreSQL storage|no|postgresql|
+|openshift_pv_size|Size of PV that's going to be created for PostgreSQL storage|no|10Gi|
```

Comment:
to be consistent, I'd recommend naming these vars with `_pg_` and `pvc` - i.e.:

```
openshift_pg_pvc_size
openshift_pg_pvc_wait_retries
openshift_pg_pvc_wait_delay
```

Reply:
Done